src/atsamd/Makefile

```diff
@@ -31,10 +32,9 @@ src-$(CONFIG_HAVE_GPIO_I2C) += atsamd/i2c.c
 src-$(CONFIG_HAVE_GPIO_SPI) += atsamd/spi.c
 src-$(CONFIG_HAVE_SERCOM) += atsamd/sercom.c
 src-$(CONFIG_HAVE_GPIO_HARD_PWM) += atsamd/hard_pwm.c
-src-$(CONFIG_MACH_SAMD21) += atsamd/watchdog.c
-src-$(CONFIG_MACH_SAMD21) += atsamd/clock.c atsamd/timer.c generic/timer_irq.c
-src-$(CONFIG_MACH_SAMX5) += atsamd/samd51_watchdog.c
-src-$(CONFIG_MACH_SAMX5) += atsamd/samd51_clock.c generic/armcm_timer.c
+src-$(CONFIG_MACH_SAMC21) += atsamd/samd51_watchdog.c atsamd/samc21_clock.c atsamd/timer.c generic/timer_irq.c
+src-$(CONFIG_MACH_SAMD21) += atsamd/watchdog.c atsamd/clock.c atsamd/timer.c generic/timer_irq.c
+src-$(CONFIG_MACH_SAMX5) += atsamd/samd51_watchdog.c atsamd/samd51_clock.c generic/armcm_timer.c
```

Comment:
It would be preferable not to mix whitespace changes with functional changes.  Can the code add the two CONFIG_MACH_SAMC21 lines without altering the existing lines?

Reply:
It can be changed back, I only did this because it seemed neater this way.